.changeset/khaki-apricots-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': patch
+---
+
+Create getProviderEnforcementState method to get reCAPTCHA Enterprise enforcement state of a provider
```

Comment:
Can you add something indicating that this is an internal code change preparing for future features, so we can be aware of that when writing release notes? Or if it will impact the user at all, or fix any bugs, add how.